pcdscalc/common.py

```diff
@@ -1,8 +1,34 @@
 """Module that holds common calculations."""
 import numpy as np
+from ophyd import EpicsSignal
 
 from .constants import WAVELENGTH_TO_ENERGY_LAMBDA, units
 
+try:
+    _energy_pv = EpicsSignal('SIOC:SYS0:ML00:AO627', name='lcls_energy')
+except NotImplementedError:
```

Comment:
I think I'm just ignorant here: in which cases does this raise a `NotImplementedError`?

Reply:
I had that error showing deep from ophyd at some point when testing. I did not really dig further besides catching it here.
Perhaps we could just have a catch-all except instead here. It's just a nice thing to have if people want to rely on the energy PV, but most of the time people will specify an energy anyway.

Reply:
I think a catch-all is completely appropriate.